### ISSUE TITLE
Fullscreen Bug on Windows 10

### ISSUE THREAD

**Issue by ditokp** [unknown-horizons/unknown-horizons#2923]
After I'm installing the game, I change the resolution to 1920 x 1080 and check the fullscreen. After the game restart it become like this:
![Unknown-Horizon-2019 1-Bug](https://user-images.githubusercontent.com/27815899/67617314-2543b100-f80c-11e9-8fe4-697b5da21a1e.png)

Re-installing the game is not fix the problem.

Specs:
Screen Resolution: 1920 x 1080

**Comment by GAFelton** [comment]
I am also having this issue. Just trying to play the game for the first time, and I cannot figure out how to fix it.

**Comment by artism90** [comment]
I'd recommend to play in window mode, fullscreen has been a constant source for trouble.
In case the game UI isn't usable in the current state, you can edit the settings.xml to disable fullscreen and change the resolution.

https://github.com/unknown-horizons/unknown-horizons/wiki/FAQ#settings

**Comment by GAFelton** [comment]
The game UI is not usable - but I am not seeing the settings.xml file. it should be in `./unknown-horizons`, right? The only file called `settings.xml` I have found via File Explorer search is this one: `unknown-horizons\content\gui\xml\mainmenu\settings.xml`

**Comment by artism90** [comment]
No, it looks like this (copy of my working configuration on Windows):
[settings.zip](https://github.com/unknown-horizons/unknown-horizons/files/5037716/settings.zip)

The settings are saved in a user/config directory, not within the game directory itself. Have you checked all the mentioned paths from the link above?
* Linux/Mac after 2017.2: `$XDG_CONFIG_HOME/unknown-horizons/settings.xml` or if this is not defined: `~/.config/unknown-horizons/settings.xml`

If it's not there, there must be something different for the Linux build I'm not aware of.

**Comment by jmdejong** [comment]
On windows the path to `settings.xml` from your home directory should be `My Games\unknown-horizons\settings.xml` (I'm not exactly sure what counts as your home directory though). If it is not there then it should be fine to put artism's settings.xml there. The file that you found contains the default settings; a settings file in your home directory will override those.

**Comment by GAFelton** [comment]
Ah! Thank you! I should have read the [location settings](https://github.com/unknown-horizons/unknown-horizons/wiki/FAQ#settings) more carefully. Your comment that it's not in the game directory helped me a lot.

**Comment by benjaminhampe** [comment]
Another consequence of missing High DPI Awareness bug. 

I have to assume the bug reporter is on Windows.

https://github.com/unknown-horizons/unknown-horizons/issues/2958

**Comment by jakoch** [comment]
This needs a bufgix on FIFE side: 
- fullscreen handling
- support hdpi
